src/components/ScholarshipDetailCard.jsx

```diff
@@ -127,25 +127,30 @@ export default function ScholarshipDetailCard({ scholarship }) {
   return (
     <Card className={classes.detailSection}>
       <Typography variant="h4">{name}</Typography>
-      <Typography variant="h5">{organization}</Typography>
-      <Box className={classes.actionSection}>
-        <Button
-          component={Link}
-          href={website}
-          variant="contained"
-          color="primary"
-          className={classes.applyBtn}
-          startIcon={<Send />}>
-          Apply
-        </Button>
-
-        <Button
-          variant="outlined"
-          className={classes.shareBtn}
-          startIcon={<Share />}
-          onClick={shareFn}>
-          Share
-        </Button>
+      <Typography variant="h5" className={review ? classes.description : ''}>
```

Comment:
Maybe add a `gutterBottom` instead?
```suggestion
      <Typography variant="h5" gutterBottom>
```

Reply:
oops, for some reason I thought the box surrounded everything lol.